aboutPage/about.js

```diff
@@ -1,3 +1,12 @@
+function show() {
+  var menu = document.getElementById("show");
+  if (menu.style.display === "none") {
```

Comment:
check the menu.style.display value for first clicking, it will give you empty 
that derive to need two clicking to display the menu